ceph-releases/ALL/centos/daemon-base/__DOCKERFILE_INSTALL__

```diff
@@ -112,5 +112,8 @@ bash -c ' \
     yum install -y dnf-plugins-core ; \
     yum copr enable -y tchaikov/python-scikit-learn ; \
     yum copr enable -y tchaikov/python3-asyncssh ; \
+    if [[ "${OSD_FLAVOR}" == "crimson" ]] ; then \
```

Comment:
The point was that crimson needed grpc/protobuf because of new code in spdk, and we know it wasn't upstream for 8stream, which is why  Ken made the copr.   I don't understand what you're saying.

Reply:
@epuertat: https://github.com/ceph/ceph-container/pull/2203

Reply:
@nizamial09 explained the saga to me.  We have three separate consumers of grpc, apparently: dashboard, nvmeof, and crimson (at least the latter because of a new version of spdk).

Reply:
Thanks for clarifying that in this comment Dan. I could not even remember all three of those.